changes/1520.bugfix.rst

```diff
@@ -0,0 +1 @@
+Added an error message when Java is used during a JDK upgrade.
```

Comment:
These messages will be compiled into release notes - so we're looking for more of a "marketing voice" than a commit message:
```suggestion
Error handling during JDK upgrades has been improved.
```